salt/utils/network.py

```diff
@@ -1958,3 +1959,34 @@ def parse_host_port(host_port):
             raise ValueError('bad hostname: "{}"'.format(host))
 
     return host, port
+
+
+@jinja_filter("filter_by_networks")
+def filter_by_networks(values, networks):
+    """
+    Returns a the list of IPs filtered by the network list.
```

Comment:
Typo: `Returns a the list...`.

Reply:
Right, that needs to be fixed. Did you have a look at the [discussion](https://github.com/saltstack/salt/pull/56394#discussion_r394536151) about an empty list vs None ? If we decide to change the meaning (from the current implementation), I need to update the docstring anyway.

Reply:
@agraul I've just left a comment on that discussion.